l2geth/eth/api_backend.go

```diff
@@ -21,6 +21,7 @@ import (
 	"errors"
 	"fmt"
 	"math/big"
+	"time"
```

Comment:
Is this `time` import required?

Reply:
`func (b *EthAPIBackend) RPCEVMTimeout() time.Duration`